services/venice-controller/src/main/java/com/linkedin/venice/controller/HelixVeniceClusterResources.java

```diff
@@ -265,6 +268,7 @@ public void refresh() {
     customizedViewRepo.refresh();
     pushMonitor.loadAllPushes();
     routersClusterManager.refresh();
+    admin.startInstanceMonitor();
```

Comment:
`HelixVeniceClusterResources` object is cluster specific, so I wonder whether this will work in a multi-cluster setup or not.

Reply:
So it will be called multiple times for each cluster, which should be fine?

Reply:
else we can keep a map of cluster->monitors and each cluster will refresh its own monitor